is_valid.json

```diff
@@ -38,6 +38,12 @@
                 "x": null
             },
             "returns": false
+        },
```

Comment:
comment about the process description above (I can only comment on diffs, not on unchanged lines):

The numbered list layout with embedded if in the second item is bit hard to understand.

proposal, something like:

>  Checks whether the specified value x is valid, which is defined based on the type of the value as follows:
> * for a numeric value: return true if value is a finite number and false otherwise. The definition ...
> * for no-data value (`null`): return false
> * any other value: return true (even for arrays only containing `null`, empty strings, string `"null"`, ...)

Reply:
I agree that it is not easy to understand. I wasn't 100% happy with the proposal though and have submitted another proposal. Is is better now?

Reply:
yes that's easier to parse

minor suggestion:

> All arrays, objects and strings are valid, regardless of their content.